src/GivingCircle.Api.DataAccess/Repositories/UserRepository.cs

```diff
@@ -0,0 +1,128 @@
+﻿using Dapper;
+using GivingCircle.Api.DataAccess.Client;
+using GivingCircle.Api.DataAccess.Responses;
+using GivingCircle.Api.Models;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GivingCircle.Api.DataAccess.Repositories
+{
+    public class UserRepository : IUserRepository
+    {
+        private readonly PostgresClient _postgresClient;
+
+        /// <summary>
+        /// Initializes an instance of the <see cref="UserRepository"/> class
+        /// </summary>
+        /// <param name="postgresClient">The postgres client</param>
+        public UserRepository(PostgresClient postgresClient)
+        {
+            _postgresClient = postgresClient;
+        }
+
+        //USER METHODS
+
+        public async Task<GetUserResponse> GetUserAsync(string userId)
+        {
+            // Object to map the parameters to the query
+            object parameters = new { User_Id = userId };
+
+            var user = await _postgresClient.QuerySingleAsync<GetUserResponse>("SELECT * FROM users WHERE user_id = @User_Id", parameters);
+
+            return user ?? null;
+        }
+
+        public async Task<bool> CreateUserAsync(User user)
+        {
+            // The string builder
+            StringBuilder queryBuilder = new();
+
+            // This represents the number of rows effected by our query
+            int createdResult;
+
+            // Construct the query
+            var query = queryBuilder
+                .Append($"INSERT INTO users")
```

Comment:
I would put spaces at the end of each line or else the query will construct as "INSERT INTO users(user_id, first_name..." when SQL expects spaces between statements. Same for all queries